src/backends/sentry_backend_inproc.c

```diff
@@ -260,6 +260,7 @@ handle_ucontext(const sentry_ucontext_t *uctx)
 
         sentry_envelope_t *envelope
             = sentry__prepare_event(options, event, NULL);
+        // TODO: flush transaction sitting in scope?
```

Comment:
related to a decision we still have to make: what happens in the case of a hard crash?
Also note that we have the breakpad and crashpad backends as well that should also execute on whatever we decide.

Reply:
for posterity, i believe we will want to try to flush and send the transaction in case a hard crash does occur, if something like that is possible.

https://github.com/getsentry/develop/issues/443 i believe tracks the work being done across all SDKs to implement this; we'll consider this to be a stretch goal for the time being though. any updates on this not being possible either due to time or technical constraints will be noted in the aforementioned issue.

i'll clean up this TODO so it provides a little bit more context and it's a little easier to find (ie searchable keyword) once we get around to implementing this.